libraries/provider_install_package.rb

```diff
@@ -0,0 +1,65 @@
+class Chef
+  # Chef Provider for installing or removing Elasticsearch from package or source
```

Comment:
The description should read `installing or removing Elasticsearch from source`.

What about putting an `@example` (YARDoc style) usage here? Or is it an antipattern in Chef land?